### PR TITLE
add timeline and funnel chart

### PR DESCRIPTION
**Pull request by SyafiqMuizz**

### CODE REVIEW

**Comment by izairiabdlatif** [comment]
@SyafiqMuizz 

> Current dataset displays the process of multiple sales channels on their product purchase conversion rates. Although, within the visualised chart below, all five categories are summed up to show the total flow of the sales process on the product purchase. Download sample data here.

The link to the second sample data isn't working. The page shows "404 page not found" error

**Comment by izairiabdlatif** [comment]
@SyafiqMuizz 
> Although, within the visualised chart below, all five categories are summed up to show the total flow of the sales process on the product purchase

Rephrase to:

```Within the visualised chart below, all five categories are summed up to show the total flow of the sales process on the product purchase```

**Comment by SyafiqMuizz** [comment]
@izairiabdlatif 

> Rephrase to:
> 
> `Within the visualised chart below, all five categories are summed up to show the total flow of the sales process on the product purchase`

> The link to the second sample data isn't working. The page shows "404 page not found" error

Both issues have been fixed and amended in the new commit

**Comment by izairiabdlatif** [comment]
@SyafiqMuizz 

> Current dataset displays the process of multiple sales channels on their product purchase conversion rates. Within the visualised chart below, all five categories are summed up to show the total flow of the sales process on the product purchase. Download sample data here.
>
> Within the dataset, there are five different fields where the conversion rates are caused through, those being Ads, Media, Affiliates, Referrals and Direct, where each of those have their own number values from top to bottom. The dataset also has the fields for Stages and Total which will be used for the creation of this chart.

Rephrase to:


```Current dataset displays the process of customer acquisition from multiple sales channels. In the visualised chart below, figures from all five sales channels (Ads, Media, Affiliates, Referrals and Direct) are summed up by each stage to show the total figures. The fields **Stage** and **Total** are used for the creation of this chart. Download sample data here.```

**Comment by SyafiqMuizz** [comment]
@izairiabdlatif 

> Rephrase to:
> 
> `Current dataset displays the process of customer acquisition from multiple sales channels. In the visualised chart below, figures from all five sales channels (Ads, Media, Affiliates, Referrals and Direct) are summed up by each stage to show the total figures. The fields **Stage** and **Total** are used for the creation of this chart. Download sample data here.`

Changes have been made to the funnel chart md file

**Comment by izairiabdlatif** [comment]
@smileEng @SyafiqMuizz 

# LGTM

File: visual/timeline-chart.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Time Line
```
Comment (tchyong) [inline visual/timeline-chart.md]: TimeLine

Comment (SyafiqMuizz) [reply visual/timeline-chart.md]: Changes have been made for Timeline Chart md

File: visual/timeline-chart.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+As the sample dataset has provided with numerous different fields, the key fields to use are **President**, **Elected Year** and **End Year**. Another field that could be used within the chart is the **Cause of Death**, as it could be placed to sort out the presidents that had different causes of death.
+
+|Bindings |Field to Select|
+|---|---|
+|Category|President|
+|Start Date|Elected Year|
+|End Date|End Year|
+|Tooltips|President|
+|Image|*(May leave blank)*|
```
[the remark this answers is not in the document]
Comment (SyafiqMuizz) [reply visual/timeline-chart.md]: Currently unsure of how to implement the ImagePath on the dataset, could I get assistance on how to do it?

[the remark this answers is not in the document]
Comment (tchyong) [reply visual/timeline-chart.md]: ``Image Path`` require string data type. 
The value for ``Image Path`` can be a valid image url OR encoded base 64 image url.

**Comment by smileEng** [comment]
@SyafiqMuizz 
For timeline please enhance with a new segment  "Analysis" (after ouput)
image


Having zoom into the above section, and PROVIDE AN ANALYSIS show casing HOW CAN THE CHART BE "READ" and perform a **BUSINESS ANALYST CONCLUSION**.



Note:
`Displayed in the chart, it can be seen that there are six different teams involved in the whole project, ranging from Product, Development, Growth, Marketing, CS and Sales. Each team had done multiple different subtasks throughout the whole project, each with having their own Start Date and End Date, making each team to reoccur in different points of the timeline.`
Is consider a BRIEF SUMMARY, not a busines analaysis analysis thus warrant a new section to DIVE DEEPER

**Comment by SyafiqMuizz** [comment]
@smileEng 
> Is consider a BRIEF SUMMARY, not a busines analayst analysis thus warrant a new section to DIVE DEEPER into the chart giving a GLIMPSE to the reader on HOW the though process of reading the chart and make conclusion for the decision makers.

Updated timeline chart documentation to have an analysis section on the Project Management use case

**Comment by tchyong** [comment]
@smileEng 
#LGTM

**Pull request merged**